### PR TITLE
Monitor general RQ queues (default, periodic and schemas)

### PR DESCRIPTION
**Pull request by rauchy**
## What type of PR is this? (check all applicable)
- [x] Feature

## Description
This PR adds queue sizes for the general RQ queues (`default`, `periodic` and `schemas`) to `/status.json`.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
image

### CODE REVIEW

File: redash/monitor.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ def get_queues():
     scheduled_queue_names = db.session.query(DataSource.scheduled_queue_name).distinct()
     query = db.session.execute(union_all(queue_names, scheduled_queue_names))
 
-    return ['celery'] + [row[0] for row in query]
+    general_rq_queues = map(lambda q: 'rq:queue:' + q, ['default', 'periodic', 'schemas'])
```
Comment (arikfr) [inline redash/monitor.py]: Why the `rq:queue:` prefix? 

Also, is there a way to ask `rq` for all the active queues?

Comment (arikfr) [reply redash/monitor.py]: Oh, is it because that's the Redis key?

Comment (rauchy) [reply redash/monitor.py]: Yupp

Comment (rauchy) [reply redash/monitor.py]: > Also, is there a way to ask rq for all the active queues?

Yeah, I refactored it and now it reports all queues (on the way, it made sense to remove the `rq:queue:` prefix due to not poking directly in Redis anymore).

File: redash/monitor.py [hunk 2]
```diff
@@ -35,14 +35,8 @@ def get_queues():
...
-    queues = {}
-
-    for queue in get_queues():
-        queues[queue] = {
-            'size': redis_connection.llen(queue)
-        }
-
-    return queues
+    return dict({queue: {'size': redis_connection.llen(queue)} for queue in get_celery_queues()}.items() +
+                {queue.name: {'size': len(queue)} for queue in Queue.all(connection=redis_connection)}.items())
```
Comment (arikfr) [inline redash/monitor.py]:
```suggestion
                {queue.name: {'size': queue.count} for queue in Queue.all(connection=redis_connection)}.items())
```

Probably better to use `Queue.count` here.

Comment (arikfr) [reply redash/monitor.py]: Just realized you're using Python's `len` and not `llen`. :man_facepalming: I guess it's my cue to go to sleep. :sleeping:

File: redash/monitor.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ def get_object_counts():
     return status
 
 
-def get_queues():
+def get_celery_queues():
```
Comment (arikfr) [inline redash/monitor.py]: This function is also used in `celery_tasks()`, so need to update the name there as well.

Comment (rauchy) [reply redash/monitor.py]: D'oh!

**Pull request merged**